### DISCUSSION TITLE
General ideas discussion page so we don't crowd the discussions page

### DISCUSSION THREAD

**Discussion by ExplodingHairyPotatoCat** [geckoking7/kgt_idle#10] [Ideas]
As the title says.

**Comment by ExplodingHairyPotatoCat** [comment]
Buff the hoe item. it's a worse version of the deagle. (Deagle stats are +3 attack, and -5% attack cooldown, while the hoe's stats are just -5% attack cooldown)

**Reply by geckoking7** [reply]
The hoe applies to all movement cooldowns

**Reply by geckoking7** [reply]
So block and that stuff

**Comment by geckoking7** [comment]
Would you like more items in the earlier zones?

**Reply by ExplodingHairyPotatoCat** [reply]
Yes, I think more early items would be a good idea, One idea I have is to make the gum and calculator from the school zone accessories that can drop, or make the rabbit also have a chance of dropping a lucky rabbits foot (which increases chance for drop rate or crit chance)

**Reply by geckoking7** [reply]
Ok, thanks for the input

**Comment by geckoking7** [comment]
How many achievements do you have?

**Reply by ExplodingHairyPotatoCat** [reply]
5 of them, and i still have -5 ap lol

**Reply by geckoking7** [reply]
Which ones you have collected

**Reply by ExplodingHairyPotatoCat** [reply]
Free achievement, 4, explosion (just got recently), you think I am stupid, xX_gamerlord69_Xx, and finally, you actually read the lore?

**Reply by geckoking7** [reply]
You got the 'You think I am stupid' lol
Well I am making the 10 zone so more content

**Reply by ExplodingHairyPotatoCat** [reply]
I think you should make a Boss zone for one of the zones. It could drop a full armor set, maybe a new move, or feature. but the main thing is that there's just one enemy that's super strong, and has some sort of gimmick. (Kinda like the enemies In the coder's lab, but preferably without the fixed amount of hits needed)

**Reply by ExplodingHairyPotatoCat** [reply]
Also yay for zone 10 :)

**Reply by geckoking7** [reply]
I try and make all new zones contain 1 normal enemy (no gimmick) 6 gimmicky bosses and a mini boss

**Reply by geckoking7** [reply]
I am probably going to add new moves sometime next week, they're coming out with book 6

**Comment by geckoking7** [comment]
I could just add a boss template fight in the last zone with stupid OP stats for you to grind for though

**Comment by ExplodingHairyPotatoCat** [comment]
You should add something to make getting kill bonuses less annoying to get, like an arena where you can fight a particular enemy repeatedly without going back into a zone, or something to allow you to stay in a zone instead of being kicked out once winning.

**Reply by geckoking7** [reply]
I would make something later in the game that drops an item which allows you to fight a specific enemy, but the kill bonuses are supposed to be painful to get, to balance their OPness.

**Reply by ExplodingHairyPotatoCat** [reply]
yeah, I guess that makes sense. Unrelated, but how do you actually get the item list? I've been fighting in the 9th zone for a while now and I feel like I'm doing something wrong

**Reply by geckoking7** [reply]
5% from file explorer thing

**Reply by ExplodingHairyPotatoCat** [reply]
Rad

**Reply by geckoking7** [reply]
Yes. What things should I work after adding book 6?

**Reply by ExplodingHairyPotatoCat** [reply]
I'd say try and finish the books

**Reply by geckoking7** [reply]
I want to have add the books to have a slight delay after zones are added since the books speed up the progress of the player while new zones increase total playtime.
4 hours of playtime for an 'idle' games is a bit not enough.

**Reply by ExplodingHairyPotatoCat** [reply]
good point, well other than the books, I think you should add a way for the wfu feature to help with the main game. either that or more zones, then go back and balance the zones (Also I think you should increase the drop chance for the item list to something like 20% because artificially increasing playtime by making things that should be there take a lot to drop)

**Reply by geckoking7** [reply]
Yeah I have plans for that, I just need ideas for what to have as day 3 of WFU

**Reply by ExplodingHairyPotatoCat** [reply]
i still haven't got day 2 yet so you have time lol

**Reply by geckoking7** [reply]
Well then...

**Comment by unc00rdinated** [comment]
Hello, I just created a github account so I could post here! It looks like there aren't a lot of ppl talking about this game... I have a few questions: How do you get Block (the first shield skill next to shield and dodge)? Where are the achievements explodinghairypotatocat was talking about? And how do I destroy the rock on the third USB?

**Reply by geckoking7** [reply]
Block is unlocked from killing water 50x.
Achievements are unlocked from a drop in zone 8.
Kill lava and it give lava.
Hope you are enjoying the game!

**Comment by ExplodingHairyPotatoCat** [comment]
Make the item list say what zones something you don't have drops from, that way you can find out what it is, because I'm never sure if i have every drop from a zone.

**Reply by geckoking7** [reply]
Ok

**Reply by geckoking7** [reply]
Should it instead be that whenever you hover over a zone it says something like 'x/y items collected'.

**Reply by ExplodingHairyPotatoCat** [reply]
oh that's smart, i never would've thought about that, I like that

**Reply by geckoking7** [reply]
:)

**Reply by unc00rdinated** [reply]
do the books tell you what items drop? cuz in the first zone it says I have 1/2, but the only things that drop according to the book are stick and then various stones

**Reply by ExplodingHairyPotatoCat** [reply]
You should've gotten a hat from the literal first kill, i think it's a gaurenteed drop

**Reply by geckoking7** [reply]
Do you think a game Dev puts effort in their games???


Sorry if there is any misunderstandings but the starter cap is a guaranteed drop after the first kill.

**Reply by geckoking7** [reply]
Oh that's a programming error, since your run started before the update that counted what item you collect, and you haven't got the item library, which sets all items that only drop once to 1.
So yes

**Reply by unc00rdinated** [reply]
I kept my starter cap, I never threw it out. What is the item library?

**Reply by geckoking7** [reply]
It something unlocked later in game

**Comment by unc00rdinated** [comment]
I feel like Dodge should be buffed, maybe it can avoid a certain number of attacks, or extend the period of time. Also you added the calculator, noice!

**Reply by geckoking7** [reply]
Dodge is vital for the 9th zone, and it becomes better in late game. I am going to technically indirectly directly buff it with kill bonuses from book 6.
Yes the calculator exists now :)

**Reply by ExplodingHairyPotatoCat** [reply]
I think it should dodge at least 1 attack because it's really hard to time due to the lack of an attack indication

**Reply by unc00rdinated** [reply]
That's actually a good idea, maybe an attack indicator for both you and the enemies...like a bar that fills up before you/they can attack again

**Reply by geckoking7** [reply]
*effort*

**Reply by geckoking7** [reply]
Well I can do that for the enemy

**Reply by ExplodingHairyPotatoCat** [reply]
It would be nice to have for the player, but at the very least a dodge or block cooldown would be nice

**Reply by geckoking7** [reply]
So like a flat number or like the border of the moves changes colour or some weird bar.

**Reply by geckoking7** [reply]
Well there is now something like it so tell me if its good

**Reply by ExplodingHairyPotatoCat** [reply]
I like the way this is done! very nice!

**Reply by geckoking7** [reply]
:D

**Reply by unc00rdinated** [reply]
It's awesome! Now I don't have to break my keyboard spamming Q and W! Also you added the next book! and a back button! wait but there is no back button on eraser

**Reply by geckoking7** [reply]
oops lol
Thanks for telling me

**Comment by unc00rdinated** [comment]
Is there a way to show the percentage of stone and item drops from the bottles?

**Reply by geckoking7** [reply]
Yes but that is unlocked in the mines

**Comment by geckoking7** [comment]
If there is anything you guys think needs a rework, or what to add next, feel free to tell me.
I am currently making a really goofy feature again (it might take a while), but after that idk what to do.

**Comment by ExplodingHairyPotatoCat** [comment]
Is there a way i can actually get the AP i'm supposed to have instead of having negative AP still? I wanna be able to use the ap upgrades but i can't (the issue that caused it was fixed, but the negative ap wasn't)

**Reply by geckoking7** [reply]
You do more achievements until it's back to the positives

**Reply by ExplodingHairyPotatoCat** [reply]
Yeah but like it's still at -10 so I don't think that's happening

**Reply by geckoking7** [reply]
Most off the green achieves are basically Easter eggs
One of which is literally a free achieve

**Reply by geckoking7** [reply]
What achievements do you have? Didn't you have -5 before?
The green achieves worth 1, Blue are 2, purple are 3 and yorange is 4

**Reply by ExplodingHairyPotatoCat** [reply]
I misread my AP because I'm stupid lol, I have 1 AP not -10 AP. I'm not sure where i got -10 from
![Screenshot 2023-09-12 9 19 42 AM](https://github.com/geckoking7/kgt_idle/assets/101602826/75dfaaae-0988-4326-8633-c5202a1d2aa4)

**Reply by geckoking7** [reply]
Yes

**Comment by unc00rdinated** [comment]
I think the stats box should use a colon instead of a dash since it might be confused with a negative sign. So instead of Hp **-** 814681246 it should be Hp **:** 814681246.

**Reply by geckoking7** [reply]
Yes, I have heard this from other people aswell

**Comment by ExplodingHairyPotatoCat** [comment]
Something that would be fun way later on (and I do mean like a lot farther in the future) would be a rebirth system :)

**Reply by ExplodingHairyPotatoCat** [reply]
Or like a shop to buy items you haven't gotten yet, like I can't for the life of me get the 8th item in the mines zone, so it would be nice to be able to buy it with xp or gold or something

**Reply by geckoking7** [reply]
I literally made the game try and avoid the rebirth system 
However maybe I can add something like this in feature 5

**Reply by ExplodingHairyPotatoCat** [reply]
I'm just kinda throwing things at the wall and seeing what seems fun lol, you don't have to do anything

**Reply by geckoking7** [reply]
What items do you have in mines, it might be a bug

**Reply by ExplodingHairyPotatoCat** [reply]
![Screenshot 2023-09-14 11 45 50 AM](https://github.com/geckoking7/kgt_idle/assets/101602826/2245ad82-9e80-441f-b820-ab8a58df0514)
these are my items.

**Reply by geckoking7** [reply]
Ok, I think your just unlucky with it lol

**Reply by ExplodingHairyPotatoCat** [reply]
Can I ask what it drops from?

**Reply by geckoking7** [reply]
Drill

**Reply by geckoking7** [reply]
Sorry but this next update will be much smaller, my computer kinda died and I have been tired from exams

**Reply by ExplodingHairyPotatoCat** [reply]
No worries, we can wait if you need a break :)

**Reply by geckoking7** [reply]
Thx mate
This really means alot :D

**Comment by ExplodingHairyPotatoCat** [comment]
> Drill


What the hell is the drop chance for it?? How unlucky am I??

**Reply by geckoking7** [reply]
2% I think, it's same as pickaxe

**Comment by ExplodingHairyPotatoCat** [comment]
What do you mean in the patch notes by "added a button to automatically go on adventure"? I can't find the button anywhere and I would like to make sure i'm not looking for something that isn't there

**Reply by geckoking7** [reply]
Oh its just a slight change in the tutorial

**Comment by ExplodingHairyPotatoCat** [comment]
Ah okay thanks

**Comment by unc00rdinated** [comment]
How do I get RAM? I have to keep playing Day 1 to shoot the eagle over and over, even after I've beaten it many times.
Also nice new update :)

**Reply by geckoking7** [reply]
Ram is collected from killing the computer.
Ok I probably make it a rarish drop WFU next update
Also thx :D

**Reply by unc00rdinated** [reply]
I found a bug-ish thing. So I just got the kill specific monster thingy, but if you turn it on without selecting anything, and then you go to that zone, you get stuck there. Maybe it should like automatically turn off after you leave the settings page if you didn't select anything, or auto select the first item on the list.

**Reply by geckoking7** [reply]
Yep, I probably should fix that.
Thx for telling tho

**Comment by ExplodingHairyPotatoCat** [comment]
Does the alternating reality of yes imply that the regular game takes place in the stationary reality no?

**Reply by geckoking7** [reply]
Possibly...

**Reply by unc00rdinated** [reply]
why is the alternating reality such a giant skill level jump? did I miss something? also what is the 'other stuff' for no grass touch? and where is the 3rd drone? I can't even remember how I got the first 2...

**Reply by geckoking7** [reply]
U haven't got the third drone so I think that's what slowing it down. It has a 1% chance of dropping when you send a drone to the school zone (it sounds low but it I've play tested this game 300 times). The no grass touch doesn't exist. I don't know why I put it there.

**Comment by unc00rdinated** [comment]
Do the books say everything that can drop? I just got Leather Pants from a meteorite, but that was not in the book.

**Reply by geckoking7** [reply]
The books should be correct. Leather pants shouldn't drop from meteorite though so it probably a bug

**Reply by ExplodingHairyPotatoCat** [reply]
What do they drop from?

**Reply by geckoking7** [reply]
The a10 (plane in American airspace)

**Reply by unc00rdinated** [reply]
oh maybe i got confused when using my fightpicker
but the pants aren't in the A10 entry either

**Reply by geckoking7** [reply]
Well then I probably forgotten to add it to the books

**Reply by unc00rdinated** [reply]
What does "DRONES_ACTIVE=2" mean

**Reply by geckoking7** [reply]
The amount of drones alive

**Reply by geckoking7** [reply]
Sorry but no update today, my computer has really killed itself.

**Reply by unc00rdinated** [reply]
rip ur computer :(
also another suggestion: is it possible to display how much health the enemy has left?

**Reply by geckoking7** [reply]
As a number? Sure

**Reply by ExplodingHairyPotatoCat** [reply]
Hope you can get your computer fixed soon! (and I do mean this as well wishes, not just so you can get back to updating.)

**Reply by geckoking7** [reply]
Thanks, it's just my dad wont let me buy a computer unless I get a really god school report next week, otherwise I'm stuck with the home computer that can't run it properly. I'll try and make a smaller update this week if I can.

**Reply by ExplodingHairyPotatoCat** [reply]
Good luck with the grades, and don't be afraid to focus on your grades instead of updating if you need to

**Reply by geckoking7** [reply]
Thanks :)

**Reply by unc00rdinated** [reply]
wish u the best, may the grades go up as fast as I kill the grass

**Reply by geckoking7** [reply]
Sorry but no update today either, however there will be one  next week

**Reply by ExplodingHairyPotatoCat** [reply]
What'd you get on your report card? did you get the computer you wanted?
I can't think of a way to say this without it sounding like I have an ulterior motive lol

**Reply by geckoking7** [reply]
I got first in the class for most classes, but my dad just, let's just say he's built very different. Don't worry though I can work on this at school. Also if if he wasn't, KGT would probably not exist.

Now for my extremely suspicious question, what country do you live, it's just that sometime I get beeped by my phone at midnight.

**Reply by ExplodingHairyPotatoCat** [reply]
I'm in the U.S.

**Reply by unc00rdinated** [reply]
congrats! it's pretty impressive you continue to work on this while having to do school stuff. and it is nighttime where I am rn so you might get weird notifs from me too

**Comment by ExplodingHairyPotatoCat** [comment]
Nice job on the update, pleasant quality of life changes, and bug fixes!

**Reply by geckoking7** [reply]
Thanks :)
I also added a timeline of stuff in the changelog

**Reply by unc00rdinated** [reply]
Yay Health indicator! Also i have an idea: something impressive should pop up when you satisfy a kill bonus

**Reply by geckoking7** [reply]
Sounds smart

**Reply by unc00rdinated** [reply]
maybe when u get a FightPicker69 it should tell you what you unlocked. it gets complicated after you have so many. also what if the items collected give you other boosts as well instead of just health? oh and sometimes WFU has a glitch where it starts in the among us game and you don't have a gun.

**Reply by unc00rdinated** [reply]
i think the WFU glitch happens when you die on day 1 and then the next time you play it starts on day 2
oh and can i shoot the emus? that would be hilarious

**Reply by geckoking7** [reply]
Ok, heres a list of things I try to do this week;
Subfeature of gaming  (Thing Ive mainly been working on)
Show kills of enemy on enemy death 
Show enemy unlocked in fight picker

For items collected having a new boost, I don't know what that could be since i'm trying to make it much harder to determine if you will win a fight (more health increases amount of moves the player can do).
WFU days are calculated to be in a random order so 'day 1' (kill eagle) can happen after 'day 2' (amogus). Its still possible however, just utilise your intellectual capabities
Also emus are bulletproof

**Reply by unc00rdinated** [reply]
*dial-up internet noises as my brain thinks* oh yeah cars (and emus) can kill ppl
no wonder they won a war against australians they are bulletproof *smacks forehead*

**Reply by geckoking7** [reply]
lol I was thinking of changing that but meh I'm going to leave that up for people themselves to think

**Comment by unc00rdinated** [comment]
WAIT NEW UPDATE????? woah it's so cool that you implemented my suggestions!
oh yeah I have a question: when u get a USB that upgrades the upgrade speed, does it affect only the drone selected or all drones? what if you haven't selected any?

**Reply by geckoking7** [reply]
The upgrade speed affects the speed of upgrading any drone. My brain can't comprehend what most of you said means but I dum.
If you suggest anything that makes some sense I'll probably implement it

**Reply by geckoking7** [reply]
Also give me feedback on the new subfeature thingy

**Comment by ExplodingHairyPotatoCat** [comment]
Great update! I still think starting without the gun on the second day after upgrading is just a punishment for playing the game and upgrading stuff, but the subfeature is cool!

**Reply by ExplodingHairyPotatoCat** [reply]
I hate the impostor so goddamn much no matter how much i upgrade my health it drains fuckin instantly (not as a bug but as an "i'm mad and need to complain about something" statement)

**Reply by ExplodingHairyPotatoCat** [reply]
I haven't completed a single day since I upgraded the damn thing so i don't get the gun

**Reply by geckoking7** [reply]
Lol

**Reply by geckoking7** [reply]
I really can't relate to your pain since I've done it so much I speedrun them in a total of 30secs with 0 damage taken

**Reply by ExplodingHairyPotatoCat** [reply]
that's why games usually have playtesters lol (I've calmed down now, i'm not sure why i was so angry)

**Reply by geckoking7** [reply]
I sorry, I made this game for only me to play, I never expected anyone else to play it

**Reply by ExplodingHairyPotatoCat** [reply]
no worries, it's your game, I didn't mean to make it seem like a massive problem. The game is still good, I'm just easily angered

**Reply by geckoking7** [reply]
Don't dare ever offend KGT idle ever again >:(

**Comment by unc00rdinated** [comment]
Yay new updates! Thanks so much for the Lore Library! Could you put some text explaining how to delete items before they get to the inventory?
WAIT NOOOOOOOOOOOOOO I JUST DELETED MY STARTER CAP AND I CANT GET IT BACK NOOOOOOOOOOOOOOOOOOOOO

**Reply by geckoking7** [reply]
Sure
Also rip cap

**Reply by unc00rdinated** [reply]
This is probably a stupid question, but which enemy is gasby? Is it Wikipedia? I know I killed him since I have the achievement to kill him without equipping dodge.

**Reply by geckoking7** [reply]
Gasby is the person on the screen. It might be an error for the achievement, so I'll check it out.

**Comment by ExplodingHairyPotatoCat** [comment]
Nice update! I really like the sprites in this section, they're easily recognizable, and the turret artwork is really good!

**Reply by geckoking7** [reply]
Thank you :D
Sorry if the development of this zone is slow, I just force myself to make artwork for an enemy before coding its fight due to me might forgetting or being to lazy to draw it. I might only release one new enemy due to requiring alot of textures this week but i'll definitely finish the zone next week (hopefully (possibly maybe)).

**Comment by ExplodingHairyPotatoCat** [comment]
man, I have 54 out of 60 items and these last 6 are eluding me. I have all of the zones except for 11 labeled with me having of all the items. what are they????
![image](https://github.com/geckoking7/kgt_idle/assets/101602826/fa0d8f1a-0307-45dd-9fed-ae44c7c24c63)

**Reply by geckoking7** [reply]
Ok sorry but forgot to change the total item number, your actually missing 7. The first four are easily the rarest items in the game but they secret though. Ok the item in the third row is something you definitely should of already got, drops from sending a drone to the school zone. The 2nd last is a drop in the new zone and the last is meant to be the thing that unlocks the lore library, drops from Wikipedia. Congrats on getting 54 though I think that is also my record.

**Comment by unc00rdinated** [comment]
I got an idea, maybe be able to combine 2 of the same item in Gaming to make a better rarity item
Cuz I just got a second Legendary Deagle and it would be a shame to recycle it

**Reply by geckoking7** [reply]
How about I make it that the base rarity drop chances change depending on how much of the better rarities you recycle? So if you where to recycle your legendary deagle, the chance get legendaries or better increase for all drops
Tell me if that sounds dum

**Reply by unc00rdinated** [reply]
Noice I like this update! Also how do you get to the lore library? I think the button disappeared, or maybe i just forgor

**Reply by geckoking7** [reply]
I am sorry but it supposed to have been unlocked by an item dropped from Wikipedia.
Also thanks :)

**Reply by unc00rdinated** [reply]
also after u kill the gamer guy, it doesn't bring you back to the map

**Reply by geckoking7** [reply]
Ok I'll check that out later

**Reply by unc00rdinated** [reply]
oh also another random idea (sorry for spamming you)
the enemy where u have to click the blue swords to attack it, what if the icons all move around the screen for a little more pizzazz

**Reply by geckoking7** [reply]
That sounds cool, I'll see how I can implement that.
Also this is literally my only social interaction today so 'spam' me all you want

**Reply by unc00rdinated** [reply]
So I just killed Wikipedia again (I have killed it before the update), but I did not get an item that unlocks the lore library. is it the Work Folder? (4th row, 3rd to last item) but it says that drops from file explorer... also the map said I have found 2 of 2 items.
also how much RAM do I need to upgrade the system for the second time?

**Reply by geckoking7** [reply]
It has a 10% DC from Wikipedia. I probably forgot to update what the map things says.
Every 8th ram will allow you to upgrade the system.

**Comment by unc00rdinated** [comment]
Let's go, new update! i like how the caves and mines are darker without light now. i'm still grinding at wikipedia to get the lore library drop. any tips? would dodge be more useful than shield?
For the coolmathgames enemy, could you use ^ instead of ** to reduce confusion?
also do the color of the square of items represent a certain type of bonus? also I noticed I can't equip two of the same type of item :(

**Reply by geckoking7** [reply]
Sorry but the 9th zones enemies are very complicated and a lack of a book makes it a lot worse. Most enemies in the ninth zone get debuffed if the player does a successful dodge. For wikipedia, a successful dodge will cause it to not attack for 2 more seconds, reset its attack cooldown and reduce the chance of the weird page thing that heals it showing up. As for CMG, the thing can't really be changed due to javascript using powers as ** instead of ^.
As for the last thing, yes.

**Comment by geckoking7** [comment]
Heads up, I have like 5 different assessments coming up in the next two weeks, aswell as trying to make WFU day three. With these both combined the next few updates will probably be very small or even non existent. If you want something small like a new item I can create that but please don't expect much.

**Reply by ExplodingHairyPotatoCat** [reply]
got it, be sure to take the time off if you need!

**Reply by geckoking7** [reply]
Sorry it took this long to reply to your feedback thing but here it is.
How much items to you expect per zone, like 8ish (one for each enemy) or like NGU where the is a 5 piece set +1/2 accessories. (This can't really be applied in the early zones since +>1 of all stats other than Hp kinda breaks balance)
I'll buff training for attack and maybe heal, but it is broken for the defensive moves. 
Do you know how fast it should go cause I can do lvl 50 in about 40 secs. 
The stickman 1 drone is somewhat useless, but killer robots far exceed feddy in later game. 
Idk what you mean about not upgrading the sniper drone, is that a bug or do you expect to upgrade health as well.
I am sorry about not enough books, I'll make them all without kill bonuses on the side of working on WFU day 3.
What is your threshold for a 'rare' item since the only I view as too rare is the'e' accessory (I don't play many games so you probably know more than me). 
I'll balance zone 10 and 11 next week to give an appropriate nerf to E.
The fightpicker is supposed to lag behind the zone your currently on by 6 zones, and so thats why they seem rare.
I sorry but I really don't know much about 'good game making', you are much wiser than I am.

**Reply by ExplodingHairyPotatoCat** [reply]
I'm pretty sure the sniper drone is a bug, if you try and level it up it just says "NaN" instead of leveling. 5% drop for things is way rare for a non boss thing. I'd say like 30 percent is good as like a base rarity and depending on how rare you want it lower or raise it from there. I'd say don't go below 10% except for items you want to be exceedingly rare. I'd say 4 items per zone at minimum, more when possible. I think a set plus accessories would be nice but obviously it's your game. if your intention is for it to be 6 zones behind then it's a bit too low, as i only have 3 in zone 5 and it's completely stopped dropping for me. what do you mean by "I can do lvl 50 in about 40 secs" because if that's with your upgrades you are obscenely far upgraded with your training stat because i've had healing on for a long ass time and it's still at level 31

**Reply by ExplodingHairyPotatoCat** [reply]
please be aware, i am not a game designer either, so some of these changes might be unnecessary, or unhelpful. if you want i could try and give feedback on any specific changes you make to see if they feel nice or they should be put back to normal.  i could also help with like writting or sprites if you want (although i'm absolute dogshit at art so they might be worse than yours, but i can write  okay!)

**Comment by unc00rdinated** [comment]
Just finally opened the door and i killed e!
found a bug tho, if u use the sandwich and die, this happens, then u can kill the enemy but ur stuck on the screen
![image](https://github.com/geckoking7/kgt_idle/assets/144076336/753135cf-f515-44d1-90df-42e91aec3956)

**Reply by ExplodingHairyPotatoCat** [reply]
are you playing on an old version? try refreshing your tab. (that makes it the current version)

**Reply by unc00rdinated** [reply]
how do I knwo which version it's on (just refreshed)

**Reply by geckoking7** [reply]
If the problem persists, press f12 and tell me of any errors. This isn't an old version since the glass thing (right window) loaded but the place that says 'stuff' should have an image there, which was added before the window. Also could you tell me of the browser you use
Sorry

**Reply by ExplodingHairyPotatoCat** [reply]
as far as i know, you don't. i just refresh it every now and again.

**Reply by geckoking7** [reply]
Top update on changelog

**Reply by ExplodingHairyPotatoCat** [reply]
i was meaning in the game itself since the changelog is a different page

**Reply by geckoking7** [reply]
I dum

**Comment by ExplodingHairyPotatoCat** [comment]
you should make a discord for the sole reason of i think it might be fun to have one

**Reply by geckoking7** [reply]
Yes but ----> I have crippling social anxiety

**Reply by ExplodingHairyPotatoCat** [reply]
same but also i can't shitpost here

**Reply by geckoking7** [reply]
Ok just wait a min

**Reply by geckoking7** [reply]
https://discord.com/invite/ZzSsEwtZ

**Comment by ExplodingHairyPotatoCat** [comment]
One thing I think I should say, since I've been critical lately, is that I think the game is cool, and you're a cool dude. Nice job on the game, I could never code anything anywhere close to this!

**Reply by geckoking7** [reply]
Thank you
Any constructive criticism you have is always fine to share

**Reply by geckoking7** [reply]
Hopefully I fixed drone 4 so you can upgrade it now

**Comment by geckoking7** [comment]
Sorry about the severe absence of update this week, however I have now got an actual computer now so excepted maybe better updates from next week.
Also if I don't respond in a while I am probably dead since my dad, mightent know of this computers existence.

**Reply by geckoking7** [reply]
Ok I'm sorry but I'm feeling a bit unmotivated (*slightly a bit underexaggated*), hopefully in two weeks I might finally finish the book 7 and maybe 8 (without bonuses) with settings to allow this too the game too be playable on phone. So if you really want this update please feel free to threaten me so I aCtUaLy ReMeMbEr pEoPlE LiKe tHiS gAmE?

**Reply by geckoking7** [reply]
And when I say threaten I mean like everyday for 2 weeks send something like a picture of the earth and the captions 'I KNOW WHERE YOU LIVE, NOW UPDATE KGT'

Please do this
It will be my social interaction for the day

**Reply by ExplodingHairyPotatoCat** [reply]
![image](https://github.com/geckoking7/kgt_idle/assets/101602826/922bb182-4e76-4de0-83cc-b0067840aa43)

**Reply by geckoking7** [reply]
Thanks for the reminder :)

**Reply by ExplodingHairyPotatoCat** [reply]
:)

**Reply by ExplodingHairyPotatoCat** [reply]
I can't think of anything funny to say so update kgt idle!

**Reply by geckoking7** [reply]
Thank you. Your existence has given me pure joy

**Reply by unc00rdinated** [reply]
I was gonna say "ahem ashkully it's *your" just to be a funny troll but u fixed it without me

**Reply by geckoking7** [reply]
Yeah I somehow foresaw that coming

**Reply by geckoking7** [reply]
Should of left it tho

**Reply by unc00rdinated** [reply]
rahh grrr update kgt idle or i'll vote u imposter

**Reply by geckoking7** [reply]
Yes sir

**Reply by unc00rdinated** [reply]
YAY UPDATE!!

**Reply by unc00rdinated** [reply]
u still want reminders?

**Reply by geckoking7** [reply]
Sure, but I am sorry that this update is also delayed. I got a lot of schoolwork and other things to do.

**Reply by unc00rdinated** [reply]
yep i feel ya, have _fun_ with school

**Reply by geckoking7** [reply]
I'll try

**Comment by geckoking7** [comment]
Sorry about the very small sized updates / lack of updates for the last few months. I am currently working on rewriting WFU code, followed by WFU day 3 and hopefully after that I can resume the normal update schedule. If any of you want something specific I can try and priotize that but the next update will probably will be in 2ish weeks.

**Comment by unc00rdinated** [comment]
man WFU is actually hard now

**Reply by geckoking7** [reply]
Is that a good or a bad thing?

**Reply by unc00rdinated** [reply]
well now i actually have to try...
the shadow dude is so fast

**Comment by geckoking7** [comment]
Sorry but no update this week :(

**Comment by geckoking7** [comment]
I am very sorry guys :(

**Comment by ExplodingHairyPotatoCat** [comment]
![image](https://github.com/user-attachments/assets/a0a33408-718d-4871-a6ea-df49ba14b46c)
hey i think i know this guy

**Reply by geckoking7** [reply]
:)
 If you any of the textures changed or changes to the actual fight just ask.
Also could you tell me what you would like the background to be
Sorry about the lack of development, but I am back and motivated this time

**Reply by ExplodingHairyPotatoCat** [reply]
no worries! just keep pestering me about making the background and I will eventually get it done, I just keep forgetting to make a background lol

**Reply by geckoking7** [reply]
Masterpieces take time

**Comment by ExplodingHairyPotatoCat** [comment]
hey, on the eraser's kill bonuses, what does "halve interval" halve?

**Reply by geckoking7** [reply]
The 4 items destroyed for 1 stone becomes 2 items destroyed for 1 stone
